frontend/public/data/cambodia/tables/2021-02-06/NSO_Camel_Price_Monthly_2021.csv

```diff
@@ -0,0 +1,157 @@
+﻿Category_MN,Category_EN,Admin_Code,Month1,Month2,Month3,Month4,Month5,Month6,Month7,Month8,Month9,Month10,Month11,Month12
```

Comment:
@wadhwamatic please remove the extra data once you've confirmed it is working as you would expect :)

Reply:
done